src/parts/Sidebar/SidebarContent/Navigation/index.tsx

```diff
@@ -138,6 +79,85 @@ const Navigation = ({
   const { t } = useTranslation();
   const { vaultClientLoaded } = useSelector((state: StoreType) => state.general);
 
+  const NAVIGATION_ITEMS = React.useMemo(() => {
+    return [
+      {
+        name: 'nav_bridge',
+        link: PAGES.BRIDGE,
+        icon: RefreshIcon,
+        disabled: true
+      },
+      {
+        name: 'nav_transfer',
+        link: PAGES.TRANSFER,
+        icon: SwitchHorizontalIcon
+      },
+      {
+        name: 'nav_transactions',
+        link: PAGES.TRANSACTIONS,
+        icon: ClipboardListIcon,
+        disabled: true
+      },
+      {
+        name: 'nav_staking',
+        link: PAGES.STAKING,
+        icon: CashIcon,
+        disabled: true
+      },
+      {
+        name: 'nav_dashboard',
+        link: PAGES.DASHBOARD,
+        icon: ChartSquareBarIcon,
+        disabled: true
+      },
+      {
+        name: 'nav_vault',
+        link: PAGES.VAULT,
+        icon: ChipIcon,
+        disabled: true,
+        hidden: !vaultClientLoaded
+      },
+      {
+        name: 'separator',
+        link: '#',
+        icon: () => null,
+        separator: true
+      },
+      {
+        name: 'nav_crowdloan',
+        link: CROWDLOAN_DOMAIN,
+        icon: CashIcon,
+        external: true,
+        // This will suppress the link on testnet
+        hidden: process.env.REACT_APP_BITCOIN_NETWORK !== 'mainnet',
```

Comment:
@tomjeatt 
Minor and out of scope but it seems like 'mainnet' is hardcoded. I think we could manage it via an TypeScript enum.

Reply:
Good idea - won't tackle this now as we'll want to get all possible values, e.g. testnet, mainnet etc. but will add a ticket to the backlog.